docs/documentation/stories/1.0-update.md

```diff
@@ -0,0 +1,503 @@
+# Angular CLI migration guide
```

Comment:
where did this new file come from?

Reply:
I noticed that it is currently in the wiki, but not in our repo. https://github.com/angular/angular-cli/wiki/stories-1.0-update